### PR TITLE
Update getting_started.rst

### PR DESCRIPTION
**Pull request by Titonus**
-settings arch=x86 v -settings arch=x86_64 when trying 32 bit binaries

### CODE REVIEW

File: getting_started.rst [hunk 1]
```diff
@@ -378,11 +378,11 @@ For example, if we have a profile with a 32-bit GCC configuration in a profile c
     We strongly recommend using :ref:`profiles` and managing them with :ref:`conan_config_install`.
 
 However, the user can always override the default profile settings in the :command:`conan install` command using the :command:`--settings`
-parameter. As an exercise, try building the Encrypter project 32-bit version:
+parameter. As an exercise, try building the Encrypter project 32-bit version, after creating a new profile (gcc_x86):
```
Comment (lasote) [inline getting_started.rst]: I agree, this part is very confusing, but I don't know if we fix it only with that change.

Comment (danimtb) [reply getting_started.rst]: I think the confusion comes from the name of the profile. Will try to push some changes here. Thanks for noticing @Titonus!

**Comment by danimtb** [comment]
Suggested changes in https://github.com/Titonus/docs/pull/1

**Comment by danimtb** [comment]
As this seems to be inactive, I will collect the changes proposed in Titonus#1 and open a new PR. Thanks anyway!

**Comment by danimtb** [comment]
closing this in favor of #1338. It was already merged! Thanks again @Titonus

**Comment by Titonus** [comment]
De nada ;-)

**Pull request closed without merging**